libnmstate/schemas/operational-state.yaml

```diff
@@ -219,6 +219,19 @@ definitions:
                     type: integer
                   stp-hairpin-mode:
                     type: boolean
+                  vlans:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        vlan-range-min:
+                          type: integer
+                        vlan-range-max:
+                          type: integer
+                        pvid:
```

Comment:
How about `use-default-pvid: True/False`?

Reply:
Check [this](https://github.com/nmstate/nmstate/pull/440#discussion_r303965117) comment.

I think the better way is to hard code the default to 0 (which means leave untagged frames untagged). If the user wants to set a pvid, it can be done via the 'pvid' attribute.

There's an ongoing discussion with the layered product(s) - currently CNV - since it seems they are not interested in the 'pvid' and 'untagged' attributes.

Reply:
From the schema, my understanding is:
```
The untagged ingress package will be tagged to `vlan-default-pvid`
when `port['pvid']` is True.
```
If so, the `use-default-pvid` could be a better name.

And I don't know what you are referring in `The simplest course of action is to not expose this setting in the model`.

Reply:
> The untagged ingress package will be tagged to `vlan-default-pvid` when `port['pvid']` is True.

No, it's not like that. Untagged frames will be tagged with the value of the default-pvid. For port that have a vlan with the 'pvid' flag set, that vlan will be used to tag the ingress frame, instead of the default one.

It overrides the default pvid.

Regarding the default pvid, I propose to not implement it, and hard-code it to 0 - meaning that by default, the untagged ingress frames will *not* be tagged.

If the user so desires, he can set the 'pvid' flag in one of the vlans, but it has to be done per port.

Reply:
Thanks for the detailed explanation.

I am you remove `vlan-default-pvid` from schema, hard-code it to 0 and let each port define it's pvid.

Another concern:

Which VLAN ID should be choose as PVID when enabled?

Reply:
0, which instructs network manager to not tag untagged frames.